src/main/groovy/com/github/spotbugs/snom/internal/SpotBugsRunnerForWorker.java

```diff
@@ -91,18 +92,36 @@ public void execute() {
           findBugs2.execute();
           if (findBugs2.getErrorCount() > 0) {
             throw new GradleException(
-                "Verification failed: SpotBugs error found: " + findBugs2.getErrorCount());
+                "Verification failed: SpotBugs error found: "
+                    + findBugs2.getErrorCount()
+                    + ". SpotBugs report can be found in "
+                    + findReportPath());
           } else if (findBugs2.getBugCount() > 0) {
             throw new GradleException(
-                "Verification failed: SpotBugs violation found: " + findBugs2.getBugCount());
+                "Verification failed: SpotBugs violation found: "
+                    + findBugs2.getBugCount()
+                    + ". SpotBugs report can be found in "
+                    + findReportPath());
           }
         }
-      } catch (Exception e) {
+      } catch (GradleException gradleException) {
```

Comment:
IMO, Use just e as the variable name for GradleException.
Do you have any concerns?

```suggestion
      } catch (GradleException gradleException) {
```

Reply:
Thanks for the suggestion @wreulicke 

Should be ready again